### ISSUE TITLE
v6: pagination not working due to realtime routing

### ISSUE THREAD

**Issue by debfx** [derhuerst/db-rest#31]
Pagination doesn't seem to be working with API v6.

For example https://v6.db.transport.rest/journeys?from=8096021&to=8000085&departure=2023-08-01 returns 3 results with the latest departure at 04:46.

Still both `earlierRef` and `laterRef` are `null` so it's not possible to do follow-up request for the next results.

API v5 https://v5.db.transport.rest/journeys?from=8096021&to=8000085&departure=2023-08-01 returns working earlierRef and laterRef.

**Comment by derhuerst** [comment]
In `hafas-client`, the HAFAS client underlying `v6.db.transport.rest` & `v5.db.transport.rest`, I have changed the default `rtMode` (routing mode of the HAFAS engine) from ["hybrid" routing to "realtime" routing](https://pastebin.com/qZ9WS3Cx): https://github.com/public-transport/hafas-client/commit/3c17678d. This change has been done in `hafas-client@6` only, so that explains why the pagination is only broken in `v6.db.transport.rest`.

[The ticket in the `hafas-client` repo](https://github.com/public-transport/hafas-client/issues/287) proposes adding an option to `hafas-clients`'s `journey()` method, which could then be exposed here in `db-rest`. Then, we should also clearly document the effects of the `realtimeRouting` option.

TLDR: blocked by https://github.com/public-transport/hafas-client/issues/287

**Comment by derhuerst** [comment]
> I have published the fix (#295) as `hafas-client@6.1.0`. 🎉

https://github.com/public-transport/hafas-client/issues/287#issuecomment-1649929463

This is unblocked now. What needs to be done here:

- expose the [`opt.routingMode` option](https://github.com/public-transport/hafas-client/blob/45610fc951bb834e1b6f09e363ee820c0b92b673/p/db/readme.md#using-the-routingmode-option) as a query parameter, [in `mapRouteParsers()`](https://github.com/derhuerst/db-rest/blob/30a56869ce3b9823a303769ab3d80faff2ea20bf/api.js#L55-L70)
- document that the query parameter exists, and link to the [`hafas-client` docs](https://github.com/public-transport/hafas-client/blob/45610fc951bb834e1b6f09e363ee820c0b92b673/p/db/readme.md#using-the-routingmode-option)
- document that the trade-off: realtime routing per default and with `routingMode= REALTIME`; working pagination with e.g. `routingMode= HYBRID `

**Comment by dancojocaru2000** [comment]
Is there work in progress for this? If not, I might try looking into it, since it's affecting my usecase.

**Comment by derhuerst** [comment]
> Is there work in progress for this?

None that I know of.
